crates/torii/core/src/executor/erc.rs

```diff
@@ -30,7 +30,7 @@ pub struct RegisterErc721TokenMetadata {
     pub query: RegisterErc721TokenQuery,
     pub name: String,
     pub symbol: String,
-    pub metadata: String,
+    pub metadata: Option<String>,
```

Comment:
What is the case when this actually happens?

Reply:
there are two cases:
- nft got burnt but we still need to process the transfers
- the metadata json stored in the contract is invalid